server-data/resources/[framework]/tcrp-wagons/shared/config.lua

```diff
@@ -0,0 +1,271 @@
+Config = {}
+
+Config.HorseInvWeight = 12000 --INVweight
+
+Config.HorseInvSlots = 12 --INVslots
+--Config.StableLocations = {
```

Comment:
why is this commented out?

Reply:
seems like the table has an empty index so it was commented out to avoid an error